semeio/jobs/design2params/design2params.py

```diff
@@ -196,6 +207,14 @@ def _read_excel(file_name, sheet_name, header=0):
             )
         ) from err
 
+    p = Path(file_name)
+    if p.suffix == ".xls":
```

Comment:
Always dangerous to depend on file suffices, but it should be ok in this context.

Reply:
I agree, and `xlrd` even has an `inspect_format`. That is not available in version 1.2.0 though, would need to invert the pinning of `xlrd` in the same commit then